api/logs/views.py

```diff
@@ -29,8 +32,24 @@ def get_log(self):
             )
         return log
 
+    def check_log_permission(self, log):
+        """
+        Cycles through nodes on log backrefs.  If user can view any of the nodes, the log can be viewed.
+        """
+        auth_user = get_user_auth(self.request)
+        log_nodes = []
+
+        for node_id in log._backrefs['logged']['node']['logs']:
+            node = get_object_or_error(Node, node_id, display_name='node')
```

Comment:
Like if the `id` is invalid, for example.  There are several potential errors- permissions, or retrieving the node itself.